### ISSUE TITLE
phoneUtil.findNumbers() does not work with two bracket groups

### ISSUE THREAD

**Issue by LukasByRuthardt** [googlei18n/libphonenumber#1012]
Country/region affected (e.g., "US"): DE, CH
Example number(s) affected ("+1 555 555-1234"): (0049)(711)123456, (0041)(71)1234567

Dear Support,
I am trying to parse valid numbers with brackets around country code and area code with phoneUtil.findNumbers(). But the method only returns an empty list. 
Greetings Luke

**Comment by lararennie** [comment]
What country are you using when you call findNumbers (as defaultRegion)? Just so we can reproduce accurately with a unit test.

**Comment by LukasByRuthardt** [comment]
I'm using "DE" and "CH" as defaultRegion. 
It works when there is a whitespace between the bracket groups, e.g. "(0049) (711)123456". But without the whitespace e.g. "(0049)(711)123456", findNumbers won't find any numbers.
Greetings Luke

**Comment by qadeemakhtar** [comment]
I am using null as Default Region, and PageSource of site as text in FindNumber Method as nf.FindNumbers(PageSource , null) bt in some case it return me Empty List even there are more than 2,3 number in pageSource like 1-800-234-3368 ,1-800-678-5779
Greeting Qadeem

**Comment by lararennie** [comment]
@qadeemakhtar yes, expected, because they aren't in international format. You need to use "US", not "null" if you expect it to find US numbers.

**Comment by qadeemakhtar** [comment]
Thanks @lararennie for your response
  I don't want to be country specific, actually  I want to find phone Number (including Local , international and tool-free)  from all over the world
Greeting Qadeem

**Comment by penmetsaa** [comment]
@LukasByRuthardt 
Thanks for providing all the details we need! This sounds like a bug.
However, metadata updates are the priority for us to ensure the data is fresh.
We will get to this as soon as we have some free time.

**Comment by penmetsaa** [comment]
@qadeemakhtar,
As the numbering plans are different for each country, the library should know the country from where these numbers are dialed in order to find / extract the phone numbers (which are in formats like national, international, e.164, etc) from sources.

In case the number is in E.164 format, country is determined by the country code present in it. 
Ex: +49 711 123456
In case the number is in national format, library do not know the country until you pass the country code.
Ex: 0 711 123456

More details in [PhoneNumberUtil.findNumbers()] (https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L2849) and [PhoneNumberUtil.parse()](https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L2772).
You can also have a look at libphonenumber’s [FALSEHOODS.md](https://github.com/googlei18n/libphonenumber/blob/master/FALSEHOODS.md)

The library is working as intended here. In future, please file a separate issue in cases where your question is not related to an existing issue.

**Comment by penmetsaa** [comment]
@qadeemakhtar,
If needed, please file separate bug with complete details of your use case.

@LukasByRuthardt 
Thanks for providing all the details we need! This sounds like a bug.
However, metadata updates are the priority for us to ensure the data is fresh.
We will get to this as soon as we have some free time.
